### PR TITLE
prefer ~/.tilt-dev over ~/.windmill, now that we don't go by tilt-dev anymore

### PR DESCRIPTION
**Pull request by nicks**
Hello @landism, @maiamcc,

Please review the following commits I made in branch nicks/tilt-dev:

4e865e3b6c2a985fc30472dc1c6f4849f44b64c5 (2020-11-06 17:16:34 -0500)
prefer ~/.tilt-dev over ~/.windmill, now that we don't go by tilt-dev anymore

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

File: pkg/dirs/dirs.go [hunk 1]
```diff
@@ -55,29 +56,29 @@ func GetWindmillDir() (string, error) {
...
+	dir, err := GetTiltDevDir()
 	if err != nil {
 		return nil, err
 	}
 
-	return &WindmillDir{dir: dir}, nil
+	return &TiltDevDir{dir: dir}, nil
 }
 
 // Create a windmill dir at an arbitrary directory. Useful for testing.
```
Comment (maiamcc) [inline pkg/dirs/dirs.go:68]:
```suggestion
// Create a ~tilt-dev dir at an arbitrary directory. Useful for testing.
```

File: pkg/dirs/dirs_test.go [hunk 2]
```diff
@@ -72,8 +75,8 @@ func setup(t *testing.T) *fixture {
 	return &fixture{t: t}
 }
 
-func (f *fixture) assertWindmillDir(expected, testCase string) {
-	actual, err := GetWindmillDir()
+func (f *fixture) assertTiltDevDir(expected, testCase string) {
+	actual, err := GetTiltDevDir()
 	if err != nil {
```
Comment (maiamcc) [inline pkg/dirs/dirs_test.go:80]: ln 91: s/windmill/tilt-dev

File: pkg/dirs/dirs_test.go [hunk 3]
```diff
@@ -8,43 +8,46 @@ import (
 	"testing"
 )
 
-func TestWindmillDir(t *testing.T) {
+func TestTiltDevDir(t *testing.T) {
 	emptyPath := ""
 	oldWmdaemonHome := os.Getenv("WMDAEMON_HOME")
 	oldHome := os.Getenv("HOME")
-	oldWindmillDir := os.Getenv("WINDMILL_DIR")
+	oldTiltDevDir := os.Getenv("WINDMILL_DIR")
```
Comment (maiamcc) [inline pkg/dirs/dirs_test.go:15]: a. should this be oldWindmillDir?
b. do you have tests for how .windmill interacts with .tilt-dev? (not persuaded you need them tho)

**Review by maiamcc — APPROVED** [review]

File: pkg/dirs/dirs.go (hunk 1 above)
Comment (nicks) [reply pkg/dirs/dirs.go]: done

File: pkg/dirs/dirs_test.go (hunk 3 above)
Comment (nicks) [reply pkg/dirs/dirs_test.go]: done, ya, there are tests for that

File: pkg/dirs/dirs_test.go (hunk 2 above)
Comment (nicks) [reply pkg/dirs/dirs_test.go:89]: done

**Pull request merged**